modules/infrastructure-moving-registry.adoc

```diff
@@ -64,6 +64,34 @@ $ oc edit config/cluster
   nodeSelector:
     node-role.kubernetes.io/infra: ""
 ----
+
+. Verify the registry Pod has been moved to the infrastructure node.
++
```

Comment:
Better add more info like : Check the pod locates which node via oc get pods -o wide -n openshift-image-registry

Reply:
Thanks, @wzheng1. Just added that step and edited the existing steps slightly. Can you take another look? Thanks!

Reply:
Thanks! ack now : )